.github/workflows/dotcom-acceptance-tests-manual.yml

```diff
@@ -0,0 +1,109 @@
+name: Dotcom Acceptance Tests (manual)
+
+on:
+  pull_request:
+    types: [labeled]
+
+jobs:
+
+  acceptance-tests-anonymous:
+    runs-on: ubuntu-latest
+    if: contains(join(github.event.pull_request.labels.*.name, ', '), 'test')
```

Comment:
Do you think it's worth making this a little stricter? It looks like a label containing `fittest` or `attested` or `testify` would trigger this. Maybe these are weird hypotheticals I'm constructing and this won't be worth thinking about in practice.

Reply:
```suggestion
    if: contains(join(github.event.pull_request.labels.*.name, ', '), 'test/')
```